lib/ui/widgets/dismissible_card.dart

```diff
@@ -0,0 +1,30 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+
+class DismissibleCard extends StatelessWidget {
+  final Widget child;
+
+  final Function(DismissDirection) onDismissed;
+
+  final Color cardColor;
+
+  final double elevation;
+
+  const DismissibleCard(
+      {Key key,
+      @required this.onDismissed,
+      @required this.child,
+      this.elevation = 1,
+      this.cardColor})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Dismissible(
+      key: UniqueKey(),
+      onDismissed: onDismissed,
+      child: Card(
+        elevation: elevation,
```

Comment:
```suggestion
        elevation: elevation,
        margin: const EdgeInsets.all(0.0),
```

Reply:
this will remove the white rectangle around the card when you long press to reorder

Reply:
Woah good call 👍 , is there a default margin non-zero margin applied to the card widget ?